roles/ansible-os-hardening/vars/main.yml

```diff
@@ -36,15 +27,12 @@ os_security_suid_sgid_whitelist: []
 os_security_suid_sgid_remove_from_unknown: false
 
 # remove packages with known issues
-os_security_packages_clean: true
+os_security_packages_clean: false
```

Comment:
Well, I read this comment here: https://github.com/hardening-io/chef-os-hardening/pull/12#issuecomment-42947018

But I just checked, in the chef-os-hardening cookbook, the variable is set to true. So I'll set it to true here, too.